contributors/design-proposals/storage/raw-block-pv.md

```diff
@@ -729,7 +728,79 @@ The usage of pod device map path is;
 
 
 
-* unspecified defaults to 'file/ext4' today for backwards compatibility and in mount_linux.go  
+* unspecified defaults to 'file/ext4' today for backwards compatibility and in mount_linux.go
+
+# Dynamically provisioning
+
+Using dynamic provisioning, user is able to create block volume via provisioners. Currently,
+we have two types of provisioners, internal provisioner and external provisioner.
+During volume creation via dynamic provisioner, user passes persistent volume claim which
+contains `volumeMode` parameter, then the persistent volume claim object is passed to
+provisioners. Therefore, in order to create block volume, provisioners need to support
+`volumeMode` and then create persistent volume with `volumeMode`.
+
+As for the external provisioner case, admin have to carefully provide Kubernetes environment
```

Comment:
Can you outline both for internal and external, what happens if the plugin does not support block?

Reply:
done.